src/bots/faq/subCommands/addFaq.js

```diff
@@ -0,0 +1,37 @@
+import Faq from '../models/Faq';
+import client from '../../../client';
+import { getFormattedFaq, getStrings } from '../constants';
+import { getMemberFromMessage, isMod } from '../../../utils/perms';
+
+/**
+ * Handles adding an faq to Faq schema and sends message with new FAQ
+ * @param {Array.<string>} args - rest of command arguments
+ */
+const handler = async (args) => {
+  if (!isMod(getMemberFromMessage())) {
+    client.message.channel.send(getStrings().insufficientPermissions);
+    return;
+  }
+
+  if (args.length < 2) {
+    client.message.channel.send(getStrings().insufficientArgumentsAddFaq);
+    return;
+  }
+
+  let newFaq = await Faq({
+    definition: args.slice(1).join(' '),
+    term: args[0]
```

Comment:
@brendacs think that we would need to create multi word terms? Some FAQs in my mind may definitely be more than one word (examples: functional programming, data structures)

Reply:
i agree, that was my thought from the beginning

Reply:
@nikgil what about simply using hyphens instead of spaces? Spaces tend to have a semantic in commands, so I think it's good to keep it to a "single-word".

E.g. `function-programming` and `data-structures`